test/Functions.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require '../functions.php';
+
+class Functions extends TestCase
+{
+    public function testDisplay_winners_success() {
+        $winners = [
+            ['prize_year' => '2020', 'author_name' => 'Douglas Stuart', 'book_title' => 'Shuggie Bain', 'author_nationality' => 'Scottish-American'],
+            ['prize_year' => '1900', 'author_name' => 'foo', 'book_title' => 'bar', 'author_nationality' => 'bar']
+        ];
+        $result = display_winners($winners);
+        $this->assertIsString($result);
+    }
+
+    public function testDisplay_winners_malformed() {
```

Comment:
And a 'failure' test to make sure your error message is returned when display_winners is given an empty array?